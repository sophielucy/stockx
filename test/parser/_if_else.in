if (x < 4) {
    print("hello world");
} else {
    print("x is greater than 4");
}
