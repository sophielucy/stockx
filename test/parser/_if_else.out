If(Binop(Id(x), Less, IntLiteral(4))) { Block(Call(print, StringLiteral(hello world))) } Else { Block(Call(print, StringLiteral(x is greater than 4))) }
